script/upgrades/MU03/MU03Checks.sol

```diff
@@ -0,0 +1,687 @@
+    );
+    require(BreakerBox(breakerBox).owner() == governance, "BreakerBox ownership not transferred to governance");
+    require(
+      MedianDeltaBreaker(medianDeltaBreaker).owner() == governance,
+      "MedianDeltaBreaker ownership not transferred to governance"
+    );
+    console2.log("Contract ownerships transferred to governance 🤝");
+  }
+
+  function verifyBiPoolManager() public view {
+    BiPoolManagerProxy bpmProxy = BiPoolManagerProxy(contracts.deployed("BiPoolManagerProxy"));
+    address bpmProxyImplementation = bpmProxy._getImplementation();
+    address expectedBiPoolManager = biPoolManager;
+    if (bpmProxyImplementation != expectedBiPoolManager) {
+      console2.log(
+        "The address of BiPoolManager from BiPoolManagerProxy: %s does not match the deployed address: %s.",
+        bpmProxyImplementation,
+        expectedBiPoolManager
+      );
+      revert("Deployed BiPoolManager does not match what proxy points to. See logs.");
+    }
+    console2.log("\tBiPoolManagerProxy has a correct implementation address 🫡");
+  }
+
+  /* ================================================================ */
+  /* =========================== Exchanges ========================== */
+  /* ================================================================ */
+
+  function verifyExchanges() public view {
+    console2.log("== Verifying exchanges... ==");
+    verifyV2Exchanges();
+    verifyTradingLimits();
+    verifyReserveFraction();
+  }
+
+  function verifyV2Exchanges() public view {
+    BiPoolManager bpm = getBiPoolManager();
+    bytes32[] memory exchanges = bpm.getExchangeIds();
+
+    // check configured pools against the config
+    if (poolConfigs.length != exchanges.length) {
+      console2.log(
+        "The number of expected pools: %s does not match the number of deployed pools: %s.",
+        poolConfigs.length,
+        exchanges.length
+      );
+      revert("Number of expected pools does not match the number of deployed pools. See logs.");
+    }
+
+    for (uint256 i = 0; i < exchanges.length; i++) {
+      bytes32 exchangeId = exchanges[i];
+      IBiPoolManager.PoolExchange memory pool = bpm.getPoolExchange(exchangeId);
+
+      // verify asset0 of the deployed pool against the config
+      if (pool.asset0 != poolConfigs[i].asset0) {
+        console2.log(
+          "The asset0 of deployed pool: %s does not match the expected asset0: %s.",
+          pool.asset0,
+          poolConfigs[i].asset0
+        );
+        revert("asset0 of pool does not match the expected asset0. See logs.");
+      }
+
+      // verify asset1 of the deployed pool against the config
+      if (pool.asset1 != poolConfigs[i].asset1) {
+        console2.log(
+          "The asset1 of deployed pool: %s does not match the expected asset1: %s.",
+          pool.asset1,
+          poolConfigs[i].asset1
+        );
+        revert("asset1 of pool does not match the expected asset1. See logs.");
+      }
```

Comment:
Let's also add the rest of the fields for shits and giggles:
- pricingModule
- spread
- referanceRateFeedID
- minimumReports
- referanceRateResetFrequency
- stablePoolResetSize